ketcher-autotests/tests/File-Management/Open-And-Save-Files/MOL-Files/mol-files.spec.ts

```diff
@@ -547,3 +549,268 @@ test('Open V3000 file with R-Groups with Fragments', async ({ page }) => {
   await openFileAndAddToCanvas('RGroup-With-Fragments.mol', page);
   await takeEditorScreenshot(page);
 });
+
+test.describe('', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
+    await waitForIndigoToLoad(page);
+  });
+
+  test.describe('', () => {
+    /*
+     * Test case: EPMLSOPKET-1852, 1856, 1874, 1890, 1895, 1979, 8914, 12966, 4731, 5259, 1875, 1876, 12963
+     * Description: The structure is correctly rendered on the canvas
+     */
+    const files = [
+      {
+        testName:
+          'Open and Save files - Open/Save structure with bond properties',
+        path: 'Molfiles-V2000/all-bond-properties.mol',
+      },
+      {
+        testName:
+          'Open and Save file - Open/Save Alias and Pseudoatoms 1/2 - open',
+        path: 'Molfiles-V2000/mol-1856-to-open.mol',
```

Comment:
Could you, please, add more meaningful names for files, describing their contents? May be some files are not even needed. Sometimes in test-cases they have different names, but their content is the same